app/src/main/res/values/strings.xml

```diff
@@ -2,16 +2,16 @@
 <resources>
 
     <!-- Applications list -->
-    <string name="info_applications_list_refreshed">Applications list refreshed</string>
-    <string name="button_refresh_list">Refresh the applications list</string>
-    <string name="error_application_not_found">The application %1$s has not been found!</string>
+    <string name="info_applications_list_refreshed">App list refreshed</string>
+    <string name="button_refresh_list">Refresh app list</string>
+    <string name="error_application_not_found">Could not find the %1$s app.</string>
 
     <!-- Favorites -->
-    <string name="info_no_favorites_yet">You can add your favorites applications here\nby clicking the menu button.</string>
-    <string name="button_manage_favorites">Manage favorites</string>
+    <string name="info_no_favorites_yet">Add your favorite apps here\nby clicking the menu button.</string>
+    <string name="button_manage_favorites">Favorites</string>
     <string name="button_select_favorites">Select favorites</string>
-    <string name="hint_favorites_order">You can modify the favorites order in the list below (use the empty areas on sides to scroll).</string>
-    <string name="error_too_many_favorites">Sorry, the number of favorites is limited to %1$d!</string>

```

Comment:
Correction OK for me (for information the favorites limit is also displayed in the selection dialog title to warn the user upfront, this message is displayed only if the number is effectively exceeded).

Since the message is displayed in an Android toast, should we keep the dot at the end?

Reply:
I am a fan, but removing since the other error message (also toast) doesn't have one now.

Reply:
We can let the dot for both messages if you think it looks better, it was only a suggestion and both are ok for me.